packages/desktop-ui-lib/src/lib/permission-manager/permission-manager.component.scss

```
@@ -155,3 +155,10 @@
     }
   }
 }
+// Button row — replaces Bootstrap d-flex flex-wrap gap-2 so the component
+// doesn't depend on the host app having Bootstrap utility classes loaded.
```

Stylelint (17.9.0)

[error] Unclosed string (CssSyntaxError)

(CssSyntaxError)